public/app/features/users/UsersTable.tsx

```diff
@@ -49,12 +50,17 @@ export const UsersTable = ({ users, orgId, onRoleChange, onRemoveUser }: Props)
             <th>Seen</th>
             <th>Role</th>
             <th style={{ width: '34px' }} />
-            <th></th>
+            <th>Synced from</th>
```

Comment:
nit: this looks a bit odd if no external auth providers are used.

Reply:
il see if i could change the wording. last take would be if we have externally synced users and show it

Reply:
changed it to "Origin" instead of "synced"